### ISSUE TITLE
PXD-988 ⁃ New endpoint for bulk queries

### ISSUE THREAD

**Issue by ProfOak** [uc-cdis/indexd#141]
Currently the `GET /index/` endpoint accepts a few URL parameters, one of them being `ids` [1]. The more UUIDs the longer the indexd query URL. Most browsers enforce a limit of ~2000 characters per URL, and there is a larger limit on most server software. Sometimes extremely large queries will result in a URL that breaks the maximum character limit causing the request to fail. My suggestion is to add a new endpoint to accept POST data in the body of an HTTP request. 

(NOTE: I am not attached to the endpoint name I describe below.)

## Current implementation

```
GET /index?start=1&limit=100&ids=id1,id2...idn
```

## Proposal

```
POST /bulk-index/list

body = {
    "start": 1,
    "limit": 100,
    "ids": ["id1", "id2", ..."idn"]
}
```

[1] https://github.com/uc-cdis/indexd/blob/master/indexd/index/blueprint.py#L48-L61

**Comment by zflamig** [comment]
Good idea. Also note that if you pass in a list to us we ignore limit so you may not need that part of it?

https://github.com/uc-cdis/indexd/blob/master/indexd/index/drivers/alchemy.py#L369

**Comment by philloooo** [comment]
Resolved by #154

**Comment by philloooo** [comment]
➤ Zachary Flamig commented:

Done in https://github.com/uc-cdis/indexd/pull/154
